code/game/objects/items/singularityhammer.dm

```diff
@@ -35,16 +33,19 @@
 /obj/item/singularityhammer/proc/on_unwield(obj/item/source, mob/user)
 	wielded = FALSE
 
-/obj/item/singularityhammer/update_icon_state()
-	icon_state = "mjollnir0"
+/obj/item/singularityhammer/equipped(mob/user, slot, initial)
+	. = ..()
+	color = "#212121"
+
+/obj/item/singularityhammer/dropped(mob/user, silent)
+	. = ..()
+	color = null

```

Comment:
Also use add/remove_atom_color wrapper here.

Reply:
The fact that add and remove color_atom swap argument order is really really sad.